TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<p>We can/should bypass manually copying the coordinates from TESSCut. We can use <code>query_criteria </code>to specify the name, radius, and TESS Observaitons instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>fixed, i hadn't updated this notebook since i learned we should only be using query_criteria</p>